e2e/watch-runner.ts

```diff
@@ -21,12 +21,11 @@ export default class WatchRunner {
     const cmd = `${this.helper.command.bitBin} watch --verbose`;
     if (this.helper.debugMode) console.log(rightpad(chalk.green('command: '), 20, ' '), cmd); // eslint-disable-line
     return new Promise((resolve, reject) => {
-      // this.watchProcess = childProcess.exec(cmd, { cwd: this.helper.scopes.localPath, detached: true });
       this.watchProcess = childProcess.exec(cmd, { cwd: this.helper.scopes.localPath });
       // @ts-ignore AUTO-ADDED-AFTER-MIGRATION-PLEASE-FIX!
       this.watchProcess.stdout.on('data', (data) => {
         if (this.helper.debugMode) console.log(`stdout: ${data}`);
-        if (data.includes(STARTED_WATCHING_MSG)) {
+        if (data.includes('')) {
```

Comment:
this will probably break the tests

Reply:
What do you mean?
How can a `if` statement, break the test?
What is the right way to do it?

Reply:
there is still some output you write to the console during the watch (might be a different string), just make sure you are checking the new string

Reply:
Sorry, what??

Reply:
Fixed.